File: Habarovsk/Bars.GkhGji.Regions.Habarovsk/Entities/SMEV2/SMEVPropertyType.cs
namespace Bars.GkhGji.Regions.Habarovsk.Entities
{
    using Bars.B4.DataAccess;
    using Bars.Gkh.Entities;
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Text;
    using System.Threading.Tasks;
    using Bars.GkhGji.Regions.BaseChelyabinsk.Enums.SMEV;
    using Enums;
    using Bars.B4.Modules.FileStorage;

    public class SMEVPropertyType : BaseEntity
    {
        /// <summary>
        /// ID сообщения в системе СМЭВ
        /// </summary>
        public virtual string MessageId { get; set; }

        /// <summary>
        /// Инициатор запроса
        /// </summary>
        public virtual Inspector Inspector { get; set; }

        /// <summary>
        /// дата запроса
        /// </summary>
        public virtual DateTime CalcDate { get; set; }

        /// <summary>
        /// RealityObject
        /// </summary>
        public virtual RealityObject RealityObject { get; set; }

        /// <summary>
        /// ИНН в запросе
        /// </summary>
        public virtual Room Room { get; set; }

        /// <summary>
        /// Кадастровый номер
        /// </summary>
        public virtual string CadastralNumber { get; set; }

        /// <summary>
        /// Уровень запроса
        /// </summary>
        public virtual PublicPropertyLevel PublicPropertyLevel { get; set; }

        /// <summary>
        /// Текущее состояние запроса
        /// </summary>
        public virtual RequestState RequestState { get; set; }

        /// <summary>
        /// Доп данные для запроса, хз че мсжет понадобиться
        /// </summary>
        public virtual string Attr1 { get; set; }
        /// <summary>
        /// Доп данные для запроса, хз че мсжет понадобиться 
        /// </summary>
        public virtual string Attr2 { get; set; }
        /// <summary>
        /// Доп данные для запроса, хз че мсжет понадобиться 
        /// </summary>
        public virtual string Attr3 { get; set; }
        /// <summary>
        /// Доп данные для запроса, хз че мсжет понадобиться 
        /// </summary>
        public virtual string Attr4 { get; set; }
        /// <summary>
        /// Доп данные для запроса, хз че мсжет понадобиться 
        /// </summary>
        public virtual string Attr5 { get; set; }

    }
}

File: Voronezh/Bars.GkhGji.Regions.Voronezh/libs/B4/model/smev2/SMEVPropertyType.js
Ext.define('B4.model.smev2.SMEVPropertyType', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'SMEVPropertyType'
    },
    fields: [
        { name: 'ReqId'},
        { name: 'RequestDate'},
        { name: 'Inspector'},
        { name: 'RequestState', defaultValue:0 },
        { name: 'RealityObject' },
        { name: 'Room' },
        { name: 'CadastralNumber' },
        { name: 'Municipality'},
        { name: 'RealityObject' },
        { name: 'PublicPropertyLevel', defaultValue: 10  },
        { name: 'Attr1' },
        { name: 'Attr2' },
        { name: 'Attr3' },
        { name: 'Attr4' },
        { name: 'Attr5' },
        { name: 'CalcDate' },
        { name: 'MessageId' },


    ]
});